update adsb traffic detection docs (#2531)
PX4/PX4-user_guide: de/advanced_features/traffic_avoidance_adsb.md

| Parameter                                                                                                           | Description                                                                                                       |
| ------------------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------- |
| <span id="NAV_TRAFF_AVOID"></span>[NAV_TRAFF_AVOID](../advanced_config/parameter_reference.md#NAV_TRAFF_AVOID)    | Enable traffic avoidance mode specify avoidance response. 0: Disable, 1: Warn only, 2: Return mode, 3: Land mode. |

> | <span id="NAV_TRAFF_A_HOR"></span>[NAV_TRAFF_A_HOR](../advanced_config/parameter_reference.md#NAV_TRAFF_A_HOR) | Set NAV TRAFFIC AVOID horizontal distance                                                              |

Reviewer: Is this a diameter or radius of the cylinder?

Reviewer: I have assumed radius

| <span id="NAV_TRAFF_A_VER"></span>[NAV_TRAFF_A_VER](../advanced_config/parameter_reference.md#NAV_TRAFF_A_VER) | Set NAV TRAFFIC AVOID vertical distance                                                            |
| <span id="NAV_TRAFF_COLL_T"></span>[NAV_TRAFF_COLL_T](../advanced_config/parameter_reference.md#NAV_TRAFF_COLL_T) | Estimated time until collision based on relative speed of traffic and UAV                                                            |

## Implementation

PX4 listens for valid transponder reports during missions.

> If a valid transponder report is received, PX4 first uses the traffic transponder information to estimate whether the traffic heading and height points towards and intersections with the airspace of the UAV. The UAV airspace consists of a surrounding cylinder defined by NAV_TRAFF_A_HOR and NAV_TRAFF_A_VER with the UAV at it's center. The traffic detector then checks if the time until intersection with the UAV airspace is below the NAV_TRAFF_COLL_T threshold based on the relative speed. If the both checks are true, the [Traffic Avoidance Failsafe](../config/safety.md#traffic-avoidance-failsafe) action is started, and the vehicle will either warn, land, or return.

Reviewer: ```suggestion
If a valid transponder report is received, PX4 first uses the traffic transponder information to estimate whether the traffic heading and height indicates there will be an intersection with the airspace of the UAV.
The UAV airspace consists of a surrounding cylinder defined by [NAV_TRAFF_A_HOR](#NAV_TRAFF_A_HOR) and [NAV_TRAFF_A_VER](#NAV_TRAFF_A_VER) with the UAV at it's center.
The traffic detector then checks if the time until intersection with the UAV airspace is below the [NAV_TRAFF_COLL_T](#NAV_TRAFF_COLL_T) threshold based on the relative speed.
If the both checks are true, the [Traffic Avoidance Failsafe](../config/safety.md#traffic-avoidance-failsafe) action is started, and the vehicle will either warn, land, or return.
```
